docs/data-management/automatic-compaction.md

```diff
@@ -104,10 +48,13 @@ The automatic compaction system uses the following syntax:
     "granularitySpec": <compaction task granularitySpec>,
     "skipOffsetFromLatest": <time period to avoid compaction>,
     "taskPriority": <compaction task priority>,
-    "taskContext": <task context>
+    "taskContext": <task context>,
+    "engine": <native|msq>
```

Comment:
`engine` can be removed from here since only native is supported with coordinator-driven compaction.